clients/java/src/main/java/io/camunda/zeebe/client/impl/command/ArgumentUtil.java

```diff
@@ -43,6 +44,12 @@ public static void ensureGreaterThan(
     }
   }
 
+  public static void ensureNotNegative(final String property, final long testValue) {
```

Comment:
🔧 Can you refactor this method and the one below ? They are basically doing the same thing

```
public static void ensureNotNegative(final String property, final long testValue) {
  ........
}

public static void ensureNotNegative(final String property, final Duration testValue) {
    ensureNotNegative(property, testValue.getSeconds())
  }
```
💭 Using `testValue.toMillis()` should be fine as well 🤔

Reply:
Sure, update to reuse the code!